cmd/livepeer/livepeer.go

```diff
@@ -191,15 +193,7 @@ func main() {
 		glog.Errorf("Error creating a new node: %v", err)
 		return
 	}
-	if *transcoder && *publicIP == "" {
-		glog.Errorf("Error - transcoder needs to specify publicIP")
-		return
-	}
-	if *transcoder && *publicIP == "" {
-		glog.Errorf("Error - transcoder needs to specify publicIP")
-		return
-	}
-	nw, err := bnet.NewBasicVideoNetwork(node, *publicIP, *port)
+	nw, err := bnet.NewBasicVideoNetwork(node, "127.0.0.1", *port)
```

Comment:
Do we default to passing in 127.0.0.1 with the expectation that with the V2 networking changes, this basicnet construct won't be used anyway so it doesn't matter that a public IP isn't set for this?

Reply:
Correct, the basicnet publicIP won't be used anymore with the v2 network. Technically this API could be cleaned up too, but I'm not sure how much further work the p2p layer will see; it depends what we do with the relay network (if anything).